src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -115,14 +116,19 @@ public void testToString() {
     }
 
     @Test
-    public void testKeyEncryption() {
+    public void testKeyEncryption() throws InvalidParametersException {
         String msg = "Test decryption";
         String decrptedMsg;
 
         decrptedMsg = StringHelper.decrypt(StringHelper.encrypt(msg));
         assertEquals(msg, decrptedMsg);
     }
 
+    @Test(expectedExceptions = {InvalidParametersException.class})
+    public void testEncryptingInvalidPlaintextThrowsException() throws InvalidParametersException {
+        StringHelper.decrypt("ABC");
```

Comment:
Any chance of using equivalence partitions and boundary values to come up with better test cases? Or 'a random string' is the only test case that we can use?